### PR TITLE
Task-53936: Modify the Icons order and alignment (desktop)

### PR DESCRIPTION
**Pull request by SaraBoutej**
Before this fix icon order in the topbar is: Search, Chat, Time tracking app (if activated), Applauncher,Notification, Edit icon (if administrator).
In this PR we try to edit the icons order  and alignment in the topbar, so the new order is :
Search
Chat
Notification
Applauncher
Time tracking app (if activated)
Edit icon (if administrator)

### CODE REVIEW

File: web/portal/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml [hunk 1]
```diff
@@ -61,17 +61,23 @@
...
       <factory-id>addonContainer</factory-id>
     </container>
     <container id="AppCenterAppLauncherPortlet" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
       <name>AppCenterAppLauncherPortlet</name>
       <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
       <factory-id>addonContainer</factory-id>
     </container>
-    <container id="NotificationTopBar" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
-      <name>NotificationTopBar</name>
+    <container id="TimeTrackerPortlet" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
```
Comment (plamarque) [inline web/portal/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml]: this  "TimeTrackerPortlet" ID should not appear in the product. This container should be a generic extension point for 3rd party addons.

CC @AzmiTouil

File: web/portal/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml [hunk 2]
```diff
@@ -61,17 +61,23 @@
       <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
       <factory-id>addonContainer</factory-id>
     </container>
-    <container id="middle-topNavigation-container" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
-      <name>middle-topNavigation-container</name>
+    <container id="ChatPortlet" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
```
Comment (plamarque) [inline web/portal/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml]: Chat is not includeed in Meeds. it's an eXo Platform feature.  We should extend this in eXo, not in Meeds.

**Review by plamarque — DISMISSED** [review]
Make sure you don't leak eXo apps in Meeds.

File: web/portal/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml (hunk 2 above)
Comment (azayati) [reply web/portal/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml]: @SaraBoutej you can use a general container id and name like for middle-topNavigation-container

File: web/portal/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml (hunk 1 above)
Comment (azayati) [reply web/portal/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml]: @SaraBoutej you can use a general container id and name like for middle-topNavigation-container

**Review by azayati — DISMISSED** [review]
see review

**Review by AzmiTouil — APPROVED** [review]
LGTM

**Pull request merged**